src/modules/network-map/registration.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { IRoute, ModuleNav, NavItem } from '../../core/interfaces';
+import ViewDashboardOutlineIcon from 'mdi-react/ViewDashboardOutlineIcon';
+import GoogleCirclesExtendedIcon from 'mdi-react/GoogleCirclesExtendedIcon';
+import DomainIcon from 'mdi-react/DomainIcon';
+import useModuleTranslation from '../../core/hooks/useModuleTranslation';
+import i18n from 'i18next';
+import enData from './translations/en.json';
+import WebIcon from 'mdi-react/WebIcon';
+import { Dashboard } from './views/Dashboard';
+import { Organizations } from './views/Organizations';
+import { Nodes } from './views/Nodes';
+
+const NETWORK_MAP_ROUTE_PREFIX = '/network-map';
```

Comment:
should this be `/namespace/:namespace/network-map`?

Reply:
If the module uses namespaces yes, but the network map doesn't